data/human/culture conversations.txt

```diff
@@ -163,6 +178,21 @@ mission "Class Trip to Port"
 
 
 
+mission "Old Defense Turrets"
+	minor
+	source
+		attributes "south"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`With an annoyingly long stopover having cnouraged you to do a little mediocre exploring, you pull the rental car into the designated return parking lot. The unexpected sound of large hydraulic motors draws your attention and you watch as a massive turret lifts up from behind a nearby barricade and turns towards the sky, its single cannon poised to fire at seemingly nothing.`
```

Comment:
Why is the player in a rental car?
```suggestion
			`With an annoyingly long stopover having encouraged you to do a little exploring, you pull the rental car into the designated return parking lot. The unexpected sound of large hydraulic motors draws your attention and you watch as a massive turret lifts up from behind a nearby barricade and turns towards the sky, its single cannon poised to fire at seemingly nothing.`
```

Reply:
Because you'd think there'd be some form of transportation rather than walking or flying.

Reply:
Returning Home in the Hai missions file has the player using a taxi.

Reply:
```suggestion
			`With an annoyingly long stopover having encouraged you to do a little exploring, you pull the rental car into the designated return parking lot. The unexpected sound of large hydraulic motors draws your attention and you watch as a massive turret lifts up from behind a nearby barricade and turns toward the sky, its single cannon poised to fire at seemingly nothing.`
```
Just adding a small correction on `towards`

Reply:
It works.